vendor/plugins/wp-meta-box/inc/meta-box.php

```diff
@@ -375,6 +376,27 @@ static function normalize( $meta_box )
 		}
 
 		/**
+		 * Get field class name
+		 *
+		 * @param array $field Field array
+		 *
+		 * @return bool|string Field class name OR false on failure
+		 */
+		static function get_class_name( $field )
```

Comment:
@mehulkaklotar It seems you are making changes within a third-party plugin, wp-meta-box, is there a justification for this?